src/player/index.ts

```diff
@@ -290,7 +296,17 @@ async function renderCurrentStep() {
 
   const showNavigation = hasPreviousStep || hasNextStep || isFinalStep;
   if (!store.isEditing && showNavigation) {
-    content += "\n\n---\n";
+    let lineAndFileInfoLabel = `This step is on line ${line} in file ${step.file}`;
```

Comment:
Might want to separate @tkimenyi 's change out of this one, so we get a clean merge?

Reply:
Oh true, I didn't intend to include that here. I'll revert it now

Reply:
Changed those few lines back to how they were. Let me know if that looks good to you